AmplifyPlugins/Analytics/AWSPinpointAnalyticsPluginTests/AWSPinpointAnalyticsPluginClientBehaviorTests.swift

```diff
@@ -36,7 +34,7 @@ class AWSPinpointAnalyticsPluginClientBehaviorTests: AWSPinpointAnalyticsPluginT
     /// When: AnalyticsPlugin.identifyUser is invoked with the user profile
     /// Then: AWSPinpoint.currentEndpoint and updateEndpoint methods are called
     ///     and Hub Analytics.identifyUser event is dispatched with the input data
-    func testIdentifyUser() {
+    func TODO_testIdentifyUser() {
```

Comment:
nit: you can use XCTSkip :) https://developer.apple.com/documentation/xctest/methods_for_skipping_tests